setup.py

```diff
@@ -1,4 +0,0 @@
-from setuptools import setup
```

Comment:
this file used in python_tests.py, but not sure if it is useful now

Reply:
I am trying if it works after the deletion.